src/lib/draw/InterventionLayer.svelte

```diff
@@ -30,15 +30,33 @@
   $: gj = addLineStringEndpoints($gjSchemeCollection);
 
   // TODO Maybe have a separate component for different modes.
-  const hideWhileEditing: FilterSpecification = [
+  const hideWhileEditing: ExpressionSpecification = [
     "!=",
-    "hide_while_editing",
+    ["get", "hide_while_editing"],
```

Comment:
I wasn't aware about https://maplibre.org/maplibre-style-spec/deprecations/. We've been accidentally using the old style in some places. Half the bugs I've been hitting have been due to mixing the old and new style, which fails silently. I'm asking on slack about getting some loud console errors for this!